instrumentation/kafka/kafka-clients/kafka-clients-0.11/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/kafkaclients/KafkaSingletons.java

```diff
@@ -75,7 +76,8 @@ public static void enhanceConfig(Map<? super String, Object> config) {
         OpenTelemetryMetricsReporter.class.getName(),
         (class1, class2) -> class1 + "," + class2);
     config.put(
-        OpenTelemetryMetricsReporter.CONFIG_KEY_OPENTELEMETRY_INSTANCE, GlobalOpenTelemetry.get());
+        OpenTelemetryMetricsReporter.CONFIG_KEY_OPENTELEMETRY_SUPPLIER,
+        new OpenTelemetrySupplier(GlobalOpenTelemetry.get()));
```

Comment:
can we use a supplier here that always returns `GlobalOpenTelemetry.get()`? that way it will "actually be serializable" in case that's a real thing for kafka config 🤷‍♂️

Reply:
There is also the library instrumentation where there is no way to restore the `OpenTelemetry` instance. I thought that we can let the reporter of this issue test it. If it is needed then he'll get a NPE and we'll know that we have to do something for both javaagent and library instrumentation.